src/main/cheffy/components/api_server.clj

```diff
@@ -41,7 +42,10 @@
       (assoc ::http/routes routes/table-routes)
       ;; here we add our interceptor to the interceptor chain to include the db component
       (cheffy-interceptors [(inject-system {:system/database database})
-                            interceptors/db-interceptor])
+                            interceptors/db-interceptor
+                            (bp/body-params)
```

Comment:
Notice new interceptor that parses http body parameters using appropriate parser based on Content-Type.